glide.yaml

```diff
@@ -0,0 +1,18 @@
+package: .
+homepage: https://github.com/HewlettPackard/oneview-golang
+import:
+- package: github.com/docker/machine
+  version: v0.8.0-rc2
+  subpackages:
+  - libmachine/log
+- package: github.com/stretchr/testify
+  version: v1.1.3
```

Comment:
tried doing something like ~v0.8 but didn't have much luck on this tonight... been trying some other options.  Is semantic versioning a strict requirement, or is there some way of getting the `v` in the version number to be ignored?